clar2wasm/src/standard/standard.wat

```diff
@@ -2532,28 +2642,71 @@
         ;; For binary representation, you can take a look at https://sha256algorithm.com/
         ;; Keep in mind that SHA-256 handles 4-byte words, but SHA-512 handles 8-byte words
         ;; SHA-256 block is of 512-bits(64 bytes) but SHA-512 block is of 1024-bits(128 bytes)
-        ;; For initial values, constants and high level understanding of SHA-512, please take a look at this rust implementation
+        ;; For initial values, constants and high level understanding of SHA-512, please take a look at this Rust implementation
         ;; https://github.com/dandyvica/sha/blob/master/src/sha512.rs
         ;; If you're interested in the paper, please take a look at https://eprint.iacr.org/2010/548.pdf
```

Comment:
Still not the correct paper.

Reply:
Replaced with [this link](https://en.wikipedia.org/wiki/SHA-2)